lms/static/sass/base/_base.scss

```diff
@@ -5,20 +5,21 @@
 //   overflow-y: scroll;
 // }
 
-html,
-body {
+html {
   font-family: $font-family-sans-serif;
   font-size: $font-size-base;
```

Comment:
`font-size-base` needs to go on `body`, not `html` right?

Reply:
Oh yes, thanks for pointing out.